packages/express-backend/models/comment-services.js

```diff
@@ -50,11 +49,14 @@ function findCommentByAuthor(author) {
 }
 
 function addComment(questionID, comment) {
-  const commentToAdd = new commentModel(comment);
-  commentToAdd.questionID = questionID;
-  findQuestionById(questionID).then((question) => {
-    question.comments.push(commentToAdd._id);
-  });
+  const date = new Date();
+  const newComment = {
+    questionID: sanitizeHtml(questionID),
+    author: sanitizeHtml(comment.author),
+    body: sanitizeHtml(comment.body),
+    date: date.toDateString(),
+  };
+  const commentToAdd = new commentModel(newComment);
```

Comment:
Comments work as intended. Thank you for fixing up any problems from my initial schema and endpoibts.

Reply:
Gotchu